app/views/posts/show.html.erb

```diff
@@ -0,0 +1,27 @@
+
+  <p><%= @post.message %></p>
+  <p>
+  <strong>Comments:</strong>
+  <br><br>
+  <% @post.comments.each do |comment| %>
+  User <%= comment.user_id %>:
```

Comment:
Maybe I'm misinterpreting what this line is for but shouldn't this be username instead? (For aesthetic reasons)

Reply:
for now we aren't dealing with user class as we don’t have anything merged yet, so for mocking reason and to display which user we have simply addressed the user id but you're absolutely right username should be displayed.
Once User class is implemted we can change to username and add other functionality as well :D

Reply:
Everyone should have access to the user class since we set the db up together and we built seeds for example users!

Reply:
Yes this should be username. We were experimenting and never changed it afterwards.